templates/master/00-master/two-node-with-fencing/files/podman-etcd.yaml

```diff
@@ -0,0 +1,5 @@
+mode: 644
```

Comment:
I think this might need to be 0644
My build from this commit shows:
```
sh-5.1# ls -l /usr/local/lib/ocf/resource.d/ocp-tnf/podman-etcd
--w----r--. 1 root root 48422 Apr 10 18:00 /usr/local/lib/ocf/resource.d/ocp-tnf/podman-etcd
```